From 2ad8c3304957a363d058f3bbac6ab16070868b5f Mon Sep 17 00:00:00 2001
From: Harald Sitter <sitter@kde.org>
Date: Mon, 7 Nov 2016 13:57:08 +0100
Subject: [PATCH] debug path

---
 autotests/testhighlighter.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/autotests/testhighlighter.cpp b/autotests/testhighlighter.cpp
index 2b2118c..c6d91a3 100644
--- a/autotests/testhighlighter.cpp
+++ b/autotests/testhighlighter.cpp
@@ -144,6 +144,7 @@ private Q_SLOTS:
                 << syntax;
         }
 
+        qDebug() << QStringLiteral(TESTBUILDDIR "/output/");
         QVERIFY(QDir().mkpath(QStringLiteral(TESTBUILDDIR "/output/")));
     }
 
-- 
2.7.4

